Source/VaRest/Private/VaRestSubsystem.cpp

```diff
@@ -5,6 +5,7 @@
 #include "VaRestDefines.h"
 #include "VaRestJsonObject.h"
 #include "VaRestSettings.h"
+#include "VaRestJsonValue.h"
```

Comment:
Please check clang-format error: https://teamcity.ufna.dev/buildConfiguration/UfnaDev_VaRest_ClangFormatCheck/496?mode=branches . All plugin files are processed and checked with clang-format-9 (built-in in MSVS 2019).

Reply:
Done